lib/openshift-rest-client.js

```diff
@@ -57,9 +61,10 @@ function getNames (resourceType) {
 const spec = JSON.parse(zlib.gunzipSync(fs.readFileSync(path.join(__dirname, 'specs', 'openshift-openapi-spec.json.gz'))));
 
 /**
-  @param {object} [settings] -
-  @returns {Promise} - Returns the Rest Client
-*/
+ * Builds the rest client based on provided or default kubernetes configuration.
+ * @param {Object} settings kubernetes configuration
```

Comment:
I think for this, would she be expanding on the settings object, since there are things that can be passed in, for example,  the auth options in a passed in config like in the advanced section on the readme

```
settings.config = {
    url: process.env.CLUSTER_URL,
    auth: {
      username: process.env.ADMIN_USERNAME,
      password: process.env.ADMIN_PASSWORD
    },
    insecureSkipTlsVerify: true
  };
```

Reply:
Thanks for the review. Docs updated :+1:

Reply:
I think i was thinking of something more like this, for expanding the different settings options,  https://github.com/nodeshift/nodeshift/blob/master/index.js#L13

Reply:
Sounds good, I'll update later similar to the nodeshift docs :+1: